common/sgx/tlsverifier.c

```diff
@@ -56,6 +59,33 @@ static oe_result_t verify_report_user_data(
     return result;
 }
 
+// Verify there is a matched claim for the public key */
+static oe_result_t verify_public_key_claim(
```

Comment:
[development guide](https://github.com/openenclave/openenclave/blob/master/docs/Community/DevelopmentGuide.md) says

> The single underscore (_ ) is reserved for local definitions (static, file-scope definitions). e.g. static oe_result_t _parse_sgx_report_body(..).


```suggestion
static oe_result_t _verify_public_key_claim(
```

Reply:
Done.